packages/ubuntu_init/lib/src/launchsession/launchsession_page.dart

```diff
@@ -0,0 +1,57 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+import 'package:ubuntu_wizard/ubuntu_wizard.dart';
+import 'package:yaru_widgets/yaru_widgets.dart';
+
+import 'launchsession_l10n.dart';
+import 'launchsession_model.dart';
+
+class LaunchSessionPage extends ConsumerWidget {
+  const LaunchSessionPage({super.key});
+
+  static Future<bool> load(WidgetRef ref) {
+    return ref.read(launchsessionModelProvider).init();
+  }
+
+  @override
+  Widget build(BuildContext context, WidgetRef ref) {
+    final l10n = LaunchsessionLocalizations.of(context);
+    return WizardPage(
+      title: YaruWindowTitleBar(
+        title: Text(l10n.storeTitle),
```

Comment:
I assume this shouldn't be using storeTitle?

Reply:
I copied the store page and removed the button for launching the store.

Reply:
I'll do the changes required to model it from the final one in gnome-initial-setup.

Reply:
Changed.